target/bin/rspamd-dkim

```diff
@@ -55,6 +55,10 @@ ${ORANGE}EXAMPLES${RESET}
     ${LWHITE}setup config dkim domain example.com${RESET}
         Generate the DKIM key for a different domain (example.com).
 
+    ${LWHITE}setup config dkim keytype ed25519 domain edward.com selector elliptic-test${RESET}
+        Generate the DKIM key using the ED25519 elliptic curve for the domain
+        edward.com and the selector elliptic-test.
```

Comment:
- Any reason to use `edward.com` instead of `example.com` placeholder?
- A selector example was already provided and probably isn't adding much here?

Revised suggestion if you agree:

```suggestion
    ${LWHITE}setup config dkim keytype ed25519 domain example.com${RESET}
        Generate a DKIM key for the domain 'example.com' but with ED25519 instead of RSA.
```

Reply:
Please don't hesitate to adjust the code; the PR has been merged with auto-merge. I used "edward.com" because of ed25519 is based on Edward's Elliptical curve; but we can stick with `example.com` and remove the selector.